ci/build.py

```diff
@@ -284,7 +480,7 @@ def script_name() -> str:
                         default=1,
                         type=int)
 
-    parser.add_argument("-c", "--cache", action="store_true",
+    parser.add_argument("--no-cache", action="store_true",
```

Comment:
Can you give a bit more detail about what this change does?  What's its relation to zombie containers?